aosptree/vendor/devices-community/gd_rpi4/drm/device.mk

```diff
@@ -0,0 +1,5 @@
+$(call inherit-product, vendor/brcm/rpi4/rpi4-vendor.mk)
+
+PRODUCT_PACKAGES += \
+    android.hardware.drm-service.clearkey \
```

Comment:
```diff
+ -.-.-.-.widevine \ ?
```

Reply:
sorry, my bad, didn't see manifest changes.

Reply:
I need to move those to gd_rpi4/drm or leave them in the vendor repo... if they stay in the vendor I should also move the Wi-Fi/BT firmware there to be consistent.

That's why I left this as a draft, we need to decide if we want L3 DRM support. If we do it is worth checking if it works on the pine phones or not. It should, L3 is software DRM.

Reply:
@rsglobal do you want to include L3 DRM for RP4? If yes, I'll clean up this draft and get rid of the /vendor/brcm/rpi4 repo

Reply:
My only concern is the legal one.

Reply:
> My only concern is the legal one.

I get it, L3 is available on Raspbian by apt (libwidevinecdm0). The question is whether we can include the android equivalent or not... I don't know how the license works

Reply:
Those libs I use come from coral-tp1a.220624.014 btw, I can always add a setup-makefiles script and include them conditionally. But does it change anything?